flink-runtime/src/main/java/org/apache/flink/runtime/rpc/taskexecutor/TaskExecutor.java

```diff
@@ -35,27 +79,634 @@
  */
 public class TaskExecutor extends RpcEndpoint<TaskExecutorGateway> {
 
-	/** The unique resource ID of this TaskExecutor */
+	private static final Logger LOG = LoggerFactory.getLogger(TaskExecutor.class);
+
+	/** Return code for critical errors during the runtime */
+	private static final int RUNTIME_FAILURE_RETURN_CODE = 2;
+
+	/** The name of the TaskManager actor */
+	private static final String TASK_MANAGER_NAME = "taskmanager";
```

Comment:
Yes, the initial reason for this is that all the inner names still called "TaskManager", so the TaskExecutor class name can easily be be renamed to "TaskManager" to replace it in final. Otherwise we should replace all the "taskexecutor" into "taskmanager" after final replace.